.rubocop.yml

```diff
@@ -5,8 +5,6 @@ inherit_from:
 AllCops:
   TargetRubyVersion: 2.3
   TargetRailsVersion: 5.1
-  Include:
-    - 'app/views/api/**/*.rabl'
```

Comment:
.rabl is included per default since quite a while and starting with rubocop 0.56 this is leading to problems.